templates/workloads/fio/workload_io.yaml

```diff
@@ -0,0 +1,10 @@
+name: fio-rand-write
```

Comment:
Can we enable those params or create a new one for readwrite?
./fio --randrepeat=1 --ioengine=libaio --direct=1 --gtod_reduce=1 --name=test --filename=test --bs=4k --iodepth=64 --size=4G --readwrite=randrw --rwmixread=75

This will create a 4 GB file, and perform 4KB reads and writes using a 75%/25% (ie 3 reads are performed for every 1 write) split within the file, with 64 operations running at a time. The 3:1 ratio is a rough approximation of your typical database.

Reply:
We have couple of choices to try with different IO params
1. Get this default and change it in test case according to the load we want and submit fio.run()
2. Fio .ini files support something called as jobs, which will be sections in the config file where we can define job1..jobn with varying IO params and just submitting that job file to fio is enough to exercise all of them.

Since this is initial implementation I tried few simple and straight forward things.

Reply:
We can take this up as follow on PR, I would like e2e test team add some more configs and try them.